src/encauth/ccm/ccm_add_aad.c

```diff
@@ -37,7 +36,7 @@ int ccm_add_aad(ccm_state *ccm,
    for (y = 0; y < adatalen; y++) {
       if (ccm->x == 16) {
          /* full block so let's encrypt it */
-         if ((err = cipher_descriptor[ccm->cipher].ecb_encrypt(ccm->PAD, ccm->PAD, &ccm->K)) != CRYPT_OK) {
```

Comment:
No reason, and it is the usual way of doing in others files.
I'll refactor this patch and rename it "use the variable 'err'".